### DISCUSSION TITLE
Alert box in the Starter Theme: Content below alert box does not render

### DISCUSSION THREAD

**Discussion by binarythistle** [thuliteio/doks#301] [Q&A]
Hi there, I'm using the Starter Theme it looks like when an alert is used, e.g.

`{{< alert text="This is an Alert" >}}`

The page content following this is not rendered? I'm just running locally on development at the moment. The Child theme works ok.

Starter Theme running locally
![image](https://user-images.githubusercontent.com/815719/118923944-b4309500-b97f-11eb-8d4c-fcc7049fe4bd.png)

Child Theme running locally
![image](https://user-images.githubusercontent.com/815719/118924026-da563500-b97f-11eb-99ce-56c343162f5f.png)

**Comment by h-enk** [comment]
Confirmed (see also: https://doks.netlify.app/docs/prologue/commands/). Looking into it.

**Reply by h-enk** [reply]
Hi @binarythistle, thanks for spotting this bug. After https://github.com/h-enk/doks/pull/235 the alert shortcode needs a terminating `/`. So e.g.:

```md
{{< alert icon="👉" text="The Tutorial is intended for novice to intermediate users." />}}
```

i.s.o.

```md
{{< alert icon="👉" text="The Tutorial is intended for novice to intermediate users." >}}
```

Will be fixed + documented.

**Reply by binarythistle** [reply]
Thanks mate - great theme and really appreciate the quick response.

**Comment by cameronelliott** [comment]
Oh darn, I wish I had found this before.
Well, at least it's not a new issue you don't know about.
#325

**Comment by h-enk** [comment]
See also: h-enk/doks#235. Added [Alerts](https://getdoks.org/docs/recipes/alerts/) page.
